src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -80,7 +80,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 "/googleSecurity/**",
                 "/place/filter/**",
                 "/restorePassword/**",
-                "/updatePassword/**"
+                "/changePassword/**"
             ).permitAll()
```

Comment:
Does unregistered user can change password? Which password?

Reply:
It's for forgot password logic, when your restore your password are not logined